src/python/pants/engine/internals/graph.py

```diff
@@ -509,7 +509,7 @@ async def transitive_dependency_mapping(request: _DependencyMappingRequest) -> _
     )
 
 
-@rule(desc="Resolve transitive targets")
+@rule(desc="Resolve transitive targets", level=LogLevel.DEBUG)
```

Comment:
This change is because:
> Determine if necessary to partition MyPy input

... became misleading after #15141. We were no longer spending a majority of time determining whether to partition: instead, we were busy computing transitive dependencies. But because transitive dependency computation wasn't marked, all of the dependency computation under `TransitiveTargets`/`CoarsenedTargets` was being blamed to that code.

After #15141, the time spent outside of `TT`/`CT` and in the body of that method dropped to 4.5s... after this change, it drops to 1.5s. But the time to compute the dependencies (~11s) was still blamed to partitioning, which was misleading.

Reply:
So: IMO, we should make this change, since computing transitive dependencies is a very significant portion of the runtime of each run, and that work is currently being blamed elsewhere (such as to test runner code, or to linters themselves).